unsubscribe-emails/src/main/scala/com/gu/Main.scala

```
@@ -65,8 +70,7 @@ class MyHandler extends Handler[EncryptedStateFunctionInput, List[UnsubscribeRes
 
   def handler(user: EncryptedStateFunctionInput, context: Context): List[UnsubscribeResult] = {
     decryptInput(user).fold(
-      error => List(UnsubscribeResult("200")), // for now we just ignore error
```

we shouldn't ignore the error